packages/framework/tests/Unit/Model/Customer/CustomerServiceTest.php

```diff
@@ -344,7 +344,6 @@ public function testGetAmendedCustomerDataByOrder()
         $this->assertSame($order->getStreet(), $customerData->billingAddressData->street);
         $this->assertSame($order->getCity(), $customerData->billingAddressData->city);
         $this->assertSame($order->getPostcode(), $customerData->billingAddressData->postcode);
-        $this->assertSame($order->getTelephone(), $customerData->billingAddressData->telephone);
```

Comment:
Not this time :) I think I spent much time on this and $customerData->userData->telephone contains 'telephone'. But $order->getTelephone() contains 'orderTelephone'. This is because with userData is processed slightly different than billingAddressData.

Reply:
ok :slightly_smiling_face: